ui/scss/component/_card.scss

```diff
@@ -153,7 +153,12 @@ $card-link-scaling: 1.1;
   top: 36%
 }
 
-
+.card__icon-featured-content {
```

Comment:
As I mentioned, CSS hacks are the only option to fix the vertical centering on FA icons, but I feel like this isn't the right file. Where should it go? `_gui.scss`? Its own component stylesheet?

Reply:
I'm confused about this adjustment. Is there a standardized adjustment that can be made to all rocket icons? Or does the adjustment need to be contextual based on the spacing of the parent element(s)?

My inclination would be to keep all icon related hack fixes in either `_icons.scss` in a clearly delineated section, or even it's own file `_icon-hacks.scss` or similar.

But if only the external component can adjust the spacing, I'm less sure about this design.

Reply:
I'm using ems, I don't think it has to be contextual. I just thought of `_icons.scss` as being basically the stylesheet direct from FA, not something we want to modify. But that makes sense enough. I'll move the style there.